spec/index.bs

```diff
@@ -1233,29 +1240,21 @@ The {{HTMLScriptElement/src}} setter steps are:
     `script`.</ins>
 1.  <ins>Set [=this=]'s [=src=] content attribute to |value|.</ins>
 
-#### Setting slot values from parser #### {#setting-slot-values-from-parser}
+#### Script children changed steps #### {#script-children-changed-steps}
 
-This document modifies the HTML parser to set the [=script text=] value when the script is created.
+This document modifies the [=children changed steps=] for {{HTMLScriptElement}} as follows:
 
-Modify the [=The text insertion mode=] algorithm as follows:
+1.  If <var ignore=''>parserChange</var> is false, set [=this=]'s [=HTMLScriptElement/is trusted=] to false.
```

Comment:
This `parserChange` is a placeholder for what we end up speccing in https://github.com/whatwg/dom/issues/1288

Reply:
Which issues, besides the one mentioned in https://github.com/w3c/trusted-types/pull/533#issue-2391021496, is this PR intended to fix?

Reply:
https://github.com/w3c/trusted-types/issues/525, and https://github.com/w3c/trusted-types/issues/507 I think should both be closable once this PR is finished alongside the SVG specific one.

Reply:
When "parserChange" is `false` and `changed by trusted sink` is `true`, couldn't still malicious code have been injected? E.g. if a trusted sink called only `someScript.innerText = someScript.innerText` that'd make the untrusted code trusted.

Reply:
> E.g. if a trusted sink called only `someScript.innerText = someScript.innerText` that'd make the untrusted code trusted.

That would only work if a default policy had sanctioned that value. Else the assignment would fail before the "trusted sink" Boolean is set